src/main/kotlin/com/plus/taxiapp/domain/MemberRepository.kt

```diff
@@ -0,0 +1,5 @@
+package com.plus.taxiapp.domain
+
+interface MemberRepository {
+    fun updateDefaultPayment(memberId: Long, type: PaymentType, paymentId: Long): Member
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline